sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/unit/tests/inputs/TestEmployeeDataSet.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.inputs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.beam.sdk.values.KV;
+import org.apache.hadoop.io.Text;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+
+public class TestEmployeeDataSet {
+  /**
```

Comment:
this comment is a javadoc for the class, yes? I think in that case it needs to be immediately above the class declaration.